keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/model/Realm.kt

```diff
@@ -84,6 +84,7 @@ data class Realm(
     val adminEventsDetailsEnabled: Boolean,
     val internationalizationEnabled: Boolean,
     val supportedLocales: List<String>,
+    val defaultLocale: String = "English",
```

Comment:
Are we sure `English` is a good default here? If you set it through the UI:
![image](https://user-images.githubusercontent.com/7859791/221166844-8e4005e7-91fa-4542-a5fd-eb1e247cf10a.png)
The value is `en`. I don't know if English also works fine.